pipelines/zap.yml

```diff
@@ -1,6 +1,6 @@
 # local usage:
 #
-#   fly set-pipeline -t lite -n -c pipelines/zap.yml -p zap --load-vars-from config/local.yml -v script-branch=`git rev-parse --abbrev-ref HEAD`
```

Comment:
Oop, I edited that for my own use.  I'll put it back.
The deleted stuff isn't really necessary if you specify your branch in config.yml...